setup.py

```diff
@@ -8,6 +8,27 @@
 
 
 python_requires = "~=3.6"
+setup_requires = ["setuptools_scm"]
+install_requires = ["typing_extensions>=3.6"]
+dev_requires = [
+    "black==19.10b0",
```

Comment:
The issue with this pinning is that we don't have an easy way to update it. Our pip-tools command had an update flag.

Reply:
Was doing some reading https://snarky.ca/clarifying-pep-518/